data-access-analysis-tools/index-nb-intro.rst

```diff
@@ -0,0 +1,141 @@
+
+**Safely Log Out of JupyterLab:**
+Use the File item in the top menu bar.
+To safely shut down a Notebook, choose Close and Shutdown Notebook.
+To safely shut down a JupyterLab server and log out of the RSP, choose Save all, Exit, and Log Out.
+
+
+.. _NB-Intro-Use-A-NB:
+
+How to Use a Jupyter Notebook
+=============================
+
+The best way to learn how to use a Jupyter Notebook is to open the first of the tutorial notebooks which are provided in all users' home directories, and also available in the `tutorial-notebooks <https://github.com/rubin-dp0/tutorial-notebooks>`_ repository in the ``rubin-dp0`` GitHub Organization (see also :ref:`Examples-DP0-1-Notebooks`). 
+
+**Executing code in a Notebook:**
+Jupyter notebooks provide "cells" within which you type either Python code or markdown language (for formatted text).
+Choose the cell to execute by clicking in it with your mouse (the cursor must be in the desired cell).
+Hold down the `shift` key and press either `enter` or `return` (depending on your keyboard type), and the contents of the cell will be executed.
+If the cell type is code, and the cell contains python code, the code will be compiled and executed.
+If the cell type is markdown, then it will be rendered upon execution to yield nicely formatted text.
+(For some handy markdown tips, see `this blog post <https://medium.com/analytics-vidhya/the-ultimate-markdown-guide-for-jupyter-notebook-d5e5abf728fd>`_ or the `relevant section from the JupyterLab documentation <https://jupyter-notebook.readthedocs.io/en/latest/examples/Notebook/Working%20With%20Markdown%20Cells.html#Markdown-Cells>`_). 
+
+.. figure:: /_static/notebook.png
+    :name: notebook_aspect
+
+    An example of what users see in the Notebook Aspect. The left panel is a file tree from which you can access your personal files, and which will come pre-populated with a handful of tutorial notebooks that users can execute on their own. The largest panel is the Jupyter notebook, where markdown and code can be entered and executed. You can also see that one can have multiple notebooks open in separate tabs.
+
+**What is a kernel?**
+In the RSP Notebook Aspect, your notebooks will be operating in a kernel that has access to the full Rubin Science Pipelines, including the "Butler" (see below) that will be your main access tool to extract images and catalogs from the DP0.1 data.
+Many standard Python libraries and modules will be available, and users can install additional Python tools they wish to use.
```

Comment:
I think a link to the appropriate documentation in nb.lsst.io would be good here